client/src/components/overview/EachThumbnail.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+class EachThumbnail extends React.Component {
```

Comment:
You could make this a Functional Component if you wanted! Let me know if you need help with the props destructuring for functional components if you decide to refactor! It's not too tough!